spec/controllers/order_items_controller_spec.rb

```diff
@@ -81,6 +81,7 @@
         end
 
         expect(assigns(:item).quantity_ordered).to eq(@max_quantity - already_tied_quantity)
+        expect(assigns(:item).errors.keys).to include(:quantity_ordered)
```

Comment:
We still need a test for flash[:errors]. 

Also, making sure the errors includes quantity_ordered seems like a model test, not a controller test.